checker/raw_result.go

```diff
@@ -65,7 +65,7 @@ type CodeReviewData struct {
 
 // ContributorsData represents contributor information.
 type ContributorsData struct {
-	Users []User
+	Users []clients.Contributor
```

Comment:
anything wrong with a contributor being a user? They are the same, some fields remain `nil`

Reply:
I didn't want to attempt combining `clients.Contributor` and `clients.User` struct in this PR. If you think removing `clients.Contributor` in favor of `clients.User` makes more sense, I can attempt that in a different PR.

Reply:
gocha. Let's try in a follow-up PR then. Unless you have reservation. Wdut?

Reply:
Nope, combining the 2 structs makes sense to me.